dbproject/src/test/java/dbproject/jdbcTests.java

```diff
@@ -0,0 +1,72 @@
+package dbproject;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+public class jdbcTests {
+	private static DatabaseUtility accesor;
+	
+	@BeforeMethod
+	public void Setup(){
+		String url = "jdbc:mysql://localhost:3306/sakila";
+		this.accesor = new DatabaseUtility(url);
+		
+	}
+	@Test
+	public void firstQuery() throws SQLException
+	{
+		Connection con = accesor.getConnection();
```

Comment:
Connecting to the db and executing queries is the responsibility of the databaseUtility. Move this code into the util.